exercises/hamming/src/example.c

```diff
@@ -4,13 +4,15 @@
 
 size_t compute(const char *lhs, const char *rhs)
 {
-   size_t lhs_len = strlen(lhs);
-   size_t rhs_len = strlen(rhs);
-   size_t count = 0;
-   size_t i = 0;
-   for (i = 0; i < lhs_len && i < rhs_len; ++i) {
-      if (lhs[i] != rhs[i]) {
-         ++count;
+   size_t count = -1;
+   if (lhs && rhs && (strlen(lhs) == strlen(rhs))) {
+      count++;
+      size_t lhs_len = strlen(lhs);
+      size_t i = 0;
```

Comment:
We are trying to use C99 where possible. This means that you can actually inline this declaration in the `for` statement:

```c
for (size_t i = 0; i < lhs_len; ++i) {
```